pkg/controller/plan/adapter/ovirt/client.go

```diff
@@ -56,7 +68,13 @@ func (r *Client) CreateSnapshot(vmRef ref.Ref) (snapshot string, err error) {
 			MustBuild(),
 	).Query("correlation_id", correlationID).Send()
 	if err != nil {
-		err = liberr.Wrap(err)
+		if strings.Contains(err.Error(), "Cannot create Snapshot") {
+			err = web.ProviderNotReadyError{
```

Comment:
ProviderNotReadyError is a bit confusing because we have the ready state - and the provider could be ready in this scenario. I'd rather call it "ConflictError" - something like we had in oVirt that differentiated invalid input from "bad-timing"

Reply:
Ack, I wanted to use the ProviderNotReadyError because it's used a lot in the plan/migration.go I just need to add another error type here
https://github.com/kubev2v/forklift/blob/b8471caa9374fee5a1eddebbd39491781e257e84/pkg/controller/plan/migration.go#L808-L816